packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/PortfolioBalance.tsx

```diff
@@ -0,0 +1,98 @@
+import { makeStyles } from '@masknet/theme'
+
+import { Typography, Divider, LinearProgress } from '@material-ui/core'
+import { formatUSDValue } from './web3/utils'
+import { useI18N } from '../../../../utils/i18n-next-ui'
+import { COLOR_SY_SENIOR_TEXT, COLOR_SY_JUNIOR_TEXT } from '../../constants'
+
+const s = makeStyles()((theme) => ({
+    progress: {
+        display: 'flex',
+        marginTop: 24,
+        '& .MuiLinearProgress-colorPrimary': {
```

Comment:
Please use constants exported from `@material-ui`:

`linearProgressClasses.barColorPrimary`

All other cases are the same